static/js/components/dashboard/CourseAction.js

```diff
@@ -87,55 +89,83 @@ export default class CourseAction extends React.Component {
     );
   }
 
+  renderDescription(text: string, className: string): React$Element<*> {
+    return text.length > 0 ?
+      <div className={className}>{text}</div> :
+      null;
+  }
+
+  renderTextDescription(text: string): React$Element<*> {
+    return this.renderDescription(text, 'description');
+  }
+
+  renderBoxedDescription(text: string): React$Element<*> {
+    return this.renderDescription(text, 'boxed description');
+  }
+
   renderPayLaterLink(): React$Element<*> {
-    return (
+    return <div>
       <a href="#">Enroll and pay later</a>
-    );
+    </div>;
   }
```

Comment:
little style nit here, you could change some of these arrow functions to be wrapped in parentheses instead of curly braces, which, for a one-line function or a function that just returns JSX, lets you skip the `return` keyword.